WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableView.swift

```diff
@@ -0,0 +1,123 @@
+import UIKit
+import WordPressShared
+
+// wrap BlogListDataSource calls to add a section for the user's info cell
+class LoginEpilogueTableView: UITableViewController {
+    var blogDataSource: BlogListDataSource
+    var blogCount: Int?
+
+    required init?(coder aDecoder: NSCoder) {
+        blogDataSource = BlogListDataSource()
+        blogDataSource.loggedIn = true
+        super.init(coder: aDecoder)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let headerNib = UINib(nibName: "LoginEpilogueSectionHeader", bundle: nil)
+        tableView.register(headerNib, forHeaderFooterViewReuseIdentifier: "SectionHeader")
+    }
+
+    /// - Note: Copied from MeViewController which I bet @koke is happy about :P
+    fileprivate func defaultAccount() -> WPAccount? {
+        let context = ContextManager.sharedInstance().mainContext
+        let service = AccountService(managedObjectContext: context)
+        let account = service.defaultWordPressComAccount()
+        return account
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return blogDataSource.numberOfSections(in:tableView) + 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if (section == 0) {
+            return 1
+        } else {
```

Comment:
No need for `else`, since the line above actually redirects the Instruction Pointer